vaadin-text-field-flow-parent/vaadin-text-field-flow/src/main/java/com/vaadin/flow/component/textfield/TextField.java

```diff
@@ -381,11 +381,19 @@ public void setRequired(boolean required) {
      * conflicts with the given {@code pattern}.
      *
      * @return the {@code preventInvalidInput} property from the webcomponent
+     *
+     * @deprecated Since 23.2, this API is deprecated.
      */
+    @Deprecated
     public boolean isPreventInvalidInput() {
         return isPreventInvalidInputBoolean();
     }
 
+    /**
+     * @deprecated Since 23.2, this API is deprecated in favor of
+     *             {@code setAllowedCharPattern(charPattern)}
+     */
+    @Deprecated
     @Override
     public void setPreventInvalidInput(boolean preventInvalidInput) {
         super.setPreventInvalidInput(preventInvalidInput);
```

Comment:
Maybe we should deprecate the method in `GeneratedVaadinTextField` as well, sort of as a reminder that it should be removed together with the other methods.

Reply:
Done.